src/main/resources/bidder-config/apacdex.yaml

```diff
@@ -0,0 +1,33 @@
+adapters:
+  apacdex:
```

Comment:
I don't see enabled property for bidder and aliases. Also compare with other bidders config, and add missing to this file